src/js/EditsQ029.jsx

```diff
@@ -0,0 +1,52 @@
+var React = require('react');
+var EditsDetail = require('./EditsDetail.jsx');
+
+var EditsQ029 = React.createClass({
+  propTypes: {
+    group: React.PropTypes.array
+  },
+
+  makeSelect: function() {
```

Comment:
@awolfe76 @wpears There is no specific reason to use the drop down. Radio button is good (as long as there are two)